.gitignore

```diff
@@ -16,4 +16,5 @@
 .factorypath
 .vscode/
 repo/
-/*.svg
\ No newline at end of file
+/*.svg
```

Comment:
not new in this pr, but we probably shouldn't be ignoring .svg files

Reply:
agree